prow/scripts/deploy_and_test_kyma.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+###
```

Comment:
I am a little bit afraid about mixing all prow scripts in one directory. Maybe we should create a separate directory for running kyma-int on vm, and call it "kyma-integration-on-debian" ?